cirq-core/cirq/protocols/kraus_protocol.py

```diff
@@ -95,7 +96,7 @@ def _has_kraus_(self) -> bool:
         """
 
 
-def _strat_kraus_from_apply_channel(val: Any) -> tuple[np.ndarray, ...] | None:
+def _strat_kraus_from_apply_channel(val: Any, atol: float) -> tuple[np.ndarray, ...] | None:
```

Comment:
It would be helpful to add mention of the new `atol` parameter in the docstring.